src/control/cmd/dmg/pool.go

```diff
@@ -223,10 +223,10 @@ func poolCreate(log logging.Logger, conns client.Connect, scmSize string,
 }
 
 // poolDestroy identified by UUID.
-func poolDestroy(log logging.Logger, conns client.Connect, uuid string, force bool) error {
+func poolDestroy(log logging.Logger, conns client.Connect, poolUUID string, force bool) error {
```

Comment:
why the rename? Seems redundant. What other uuid would we be referring to in poolDestroy?

Reply:
it shadows the import identifier